### DISCUSSION TITLE
Help available?

### DISCUSSION THREAD

**Discussion by ruud00000** [vite-pwa/vite-plugin-pwa#651] [Q&A] [answered]
It looks like there are no discussions receiving any answers on this repo. 

What is the best place to ask for support using vite-plugin-pwa?

**Comment by userquin** [comment, marked as answer]
https://discord.gg/W2mebb4A

**Reply by cleangone** [reply]
Hello - this discord invite has expired - do you have another or the name of the conversation?

**Reply by userquin** [reply]
https://discord.com/invite/nVZmPTcf

**Reply by cleangone** [reply]
Thanks!

**Reply by userquin** [reply]
discord link in docs should now work, added link  without expiration
